src/lib.rs

```diff
@@ -172,3 +167,13 @@ pub trait UncompressedEncoding: Sized {
     /// the point at infinity.
     fn to_uncompressed(&self) -> Self::Uncompressed;
 }
+
+/// Obtain or determine the identity point.
+pub trait Identity: Sized {
+    /// Returns the additive identity.
+    const IDENTITY: Self;
+
+    /// Determines if this point represents the point at infinity; the
+    /// additive identity.
```

Comment:
Bug: this documentation was copied from the documentation for either `PrimeCurveAffine` or `CofactorCurveAffine`. However, for a general cryptographic group the "point" representation is not visible, and the actual curve may not have a "point at infinity" (e.g. edwards25519), or "point" may not even make sense (e.g. ristretto255).

It should instead follow the documentation from `Group` (which also had a bug in that it referred to "point" instead of "element"):
```suggestion
    /// Determines if this element is the additive identity.
```